### PR TITLE
fix ChangeLog 1.11-rc2

### PR DESCRIPTION
**Pull request by szyrom**
This change is [Reviewable](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902)

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -10,10 +10,15 @@ Fri Sep 25 2020 Szymon Romik <szymon.romik@intel.com>
 
 	New features:
 	- experimental radix_tree container (single-threaded sorted map)
-	- experimental inline_string class
-	- string_view class
+	- experimental inline_string class (class serves similar purpose to
+		pmem::obj::string, but keeps the data within the same allocation
+		as inline_string itself)
+	- string_view class (support for compilers older thatn C++17)
```
Comment (szyrom) [inline ChangeLog:16]: s/that/than

**Review by lukaszstolarczuk — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @szyrom)

___
*[ChangeLog, line 32 at r1](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902#-MIOM5Ph0-4fHXwcef5R:-MIOM5PiB4L7NB4Fy1wg:bvj64fc) ([raw file](https://github.com/pmem/libpmemobj-cpp/blob/8026f15bffff3873fe861f051f13444fc5d668e5/ChangeLog#L32)):*
> ```
> 	- lower and lower_eq methods for concurrent_map
> 	- missing constructors for concurrent_map
> ```

didn't we have any major bug fixes?

**Review by igchor — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902)* status: 0 of 1 files reviewed, 4 unresolved discussions (waiting on @szyrom)

___
*[ChangeLog, line 18 at r1](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902#-MIORVFc-cquCHrVK1xS:-MIORVFc-cquCHrVK1xT:brif54m) ([raw file](https://github.com/pmem/libpmemobj-cpp/blob/8026f15bffff3873fe861f051f13444fc5d668e5/ChangeLog#L18)):*
> ```
> 	- string_view class (support for compilers older thatn C++17)
> 	- experimental self_relative_ptr and std::atomic<self_relative_ptr>
> 		specialization (smart ptr which encapsulates the self offseted
> ```

maybe we should add that this is a **persistent** pointer?
___
*[ChangeLog, line 31 at r1](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902#-MIORczTETCwRK2VeYHT:-MIORczTETCwRK2VeYHU:b-z930t4) ([raw file](https://github.com/pmem/libpmemobj-cpp/blob/8026f15bffff3873fe861f051f13444fc5d668e5/ChangeLog#L31)):*
> ```
> 	- ported libcxx's sorted map tests
> 	- lower and lower_eq methods for concurrent_map
> 	- missing constructors for concurrent_map
> ```

which ctors?

**Review by szyrom — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902)* status: 0 of 1 files reviewed, 4 unresolved discussions (waiting on @igchor and @lukaszstolarczuk)

___
*[ChangeLog, line 18 at r1](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902#-MIORVFc-cquCHrVK1xS:-MIOf7cp8NF7rKNtP6A4:b-896fix) ([raw file](https://github.com/pmem/libpmemobj-cpp/blob/8026f15bffff3873fe861f051f13444fc5d668e5/ChangeLog#L18)):*
<details><summary><i>Previously, igchor (Igor Chorążewicz) wrote…</i></summary><blockquote>

maybe we should add that this is a **persistent** pointer?
</blockquote></details>

Done.
___
*[ChangeLog, line 31 at r1](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902#-MIORczTETCwRK2VeYHT:-MIOf6r3AVKUa5PowTfR:b-896fix) ([raw file](https://github.com/pmem/libpmemobj-cpp/blob/8026f15bffff3873fe861f051f13444fc5d668e5/ChangeLog#L31)):*
<details><summary><i>Previously, igchor (Igor Chorążewicz) wrote…</i></summary><blockquote>

which ctors?
</blockquote></details>

Done.
___
*[ChangeLog, line 32 at r1](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902#-MIOM5Ph0-4fHXwcef5R:-MIOf9-fBhrt4EwZGI5u:b-iocyhv) ([raw file](https://github.com/pmem/libpmemobj-cpp/blob/8026f15bffff3873fe861f051f13444fc5d668e5/ChangeLog#L32)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

didn't we have any major bug fixes?
</blockquote></details>

I've double-checked and didn't notice any major bugfixes.. Are there any?

**Review by igchor — COMMENTED** [review]
lgtm

> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902#-:-MIOm1Bv4CJ6Ss-J4CpZ:bb7k0qh)* status: all files reviewed, 2 unresolved discussions (waiting on @lukaszstolarczuk)

___
*[ChangeLog, line 26 at r2](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902#-MIOloPR18GVHEE1GRW4:-MIOloPS7vX4Zmj3RJ1v:b-xi206z) ([raw file](https://github.com/pmem/libpmemobj-cpp/blob/59854274376fd6a1dc0fc19046a194d32651f842/ChangeLog#L26)):*
> ```
> 	- optimizations for concurrent_map with self_relative_ptr
> 
> 	Other changes:
> ```

we can also add info that we now have no warnings on windows (for W2) :d

**Review by szyrom — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @igchor and @lukaszstolarczuk)

___
*[ChangeLog, line 26 at r2](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902#-MIOloPR18GVHEE1GRW4:-MIP1kcI8iHoAnBVmd4e:b-7m4pg0) ([raw file](https://github.com/pmem/libpmemobj-cpp/blob/59854274376fd6a1dc0fc19046a194d32651f842/ChangeLog#L26)):*
<details><summary><i>Previously, igchor (Igor Chorążewicz) wrote…</i></summary><blockquote>

we can also add info that we now have no warnings on windows (for W2) :d
</blockquote></details>

no :P

**Review by lukaszstolarczuk — APPROVED** [review]
> Reviewed 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902)* status: all files reviewed, 1 unresolved discussion

___
*[ChangeLog, line 32 at r1](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902#-MIOM5Ph0-4fHXwcef5R:-MIP3w6FA90psOdI6lLJ:b-ciey3m) ([raw file](https://github.com/pmem/libpmemobj-cpp/blob/8026f15bffff3873fe861f051f13444fc5d668e5/ChangeLog#L32)):*
<details><summary><i>Previously, szyrom (Szymon Romik) wrote…</i></summary><blockquote>

I've double-checked and didn't notice any major bugfixes.. Are there any?
</blockquote></details>

ok, thanks

**Review by igchor — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/pmem/libpmemobj-cpp/902#-:-MIPI-ib5IB2v8eU5v9E:bnfp4nl)* status: all files reviewed, 1 unresolved discussion

**Pull request merged**